ci/lint.sh

```diff
@@ -2,6 +2,8 @@
 
 set -e
 
+DART="$(pwd)/third_party/dart/tools/sdks/dart-sdk/bin/dart"
```

Comment:
Is there any way to get the directory of the script and use a path relative to that instead of `$(pwd)`?

Reply:
Done.